MAUI/NavigationDrawer/Toggle-Animations.md

```diff
@@ -0,0 +1,118 @@
+---
+layout: post
+title: Setting Toggle Animations in .NET MAUI Navigation Drawer | Syncfusion
+description: Learn here all about Setting Toggle Animations support in Syncfusion .NET MAUI Navigation Drawer (SfNavigationDrawer) control and more.
+platform: maui
+control: NavigationDrawer
+documentation: ug
+---
+# Setting Toggle Animations in .NET MAUI Navigation Drawer
+
+The drawer toggling animation can be changed using `Transition` property and it can be set to three different values. They are
+
+* `SlideOnTop`
+
+* `Push`
+
+* `Reveal`
+
+N> The default animation is `SlideOnTop`.
+
+## SlideOnTop
+
+The navigation pane overlays the main content area when it is opened. It can be set as shown below:
```

Comment:
The navigation pane overlays the main content area when it is opened. It can be set as follows:

Reply:
Changed